framework/shortcuts/ishortcutsregister.h

```diff
@@ -33,6 +33,7 @@ class IShortcutsRegister : MODULE_EXPORT_INTERFACE
     virtual ~IShortcutsRegister() = default;
 
     virtual const std::list<Shortcut>& shortcuts() const = 0;
+    virtual Shortcut shortcut(const std::string& action) const = 0;
```

Comment:
Better to rename action to actionName

Reply:
I think this is not better, because in the Shortcut interface the action has no 'name'

Reply:
renamed